src/main/java/com/sse/ooseproject/models/Person.java

```diff
@@ -28,10 +31,13 @@ protected Person(String firstName, String lastName, String email) {
         this.email = email;
     }
 
-    //Methods
+    //Method
     public long getId() {
         return this.id;
     }
+    public void setId(long id){
```

Comment:
setId muss glaub raus

Reply:
ich glaube die ist notwendig, damit die Student Instanz in der POST Methode des /student/edit Endpunktes eine gültige ID beinhaltet. Die brauche ich für den Vergleich mit der in der Datenbank vorhandenen Instanz